components/RadioButton/RadioButton.module.css

```diff
@@ -0,0 +1,79 @@
+.wrapper {
+  position: relative;
+}
+
+.input {
+  position: absolute;
+  overflow: hidden;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  border: 0;
+  white-space: nowrap;
+  clip: rect(0 0 0 0);
+}
+
+.icon {
+  width: 14px;
+  height: 14px;
+  margin-right: var(--m-0-5);
+}
+
+.label {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  height: 32px;
+  padding: 0 var(--m-2);
+  font-size: var(--fs-text-sm);
+  font-weight: 600;
+  border-radius: var(--r-default);
+  transition: background-color var(--t-interaction),
+    border-color var(--t-interaction);
+  cursor: pointer;
+
+  &.variant-gray {
+    color: var(--color-gray);
+    background-color: var(--color-white);
+  }
+
+  &.variant-green {
+    color: var(--color-green);
+    background-color: var(--color-white);
+  }
+
+  &.variant-blue {
+    color: var(--color-light-blue);
+    background-color: var(--color-white);
+  }
+}
+
+.input:hover + .label {
+  &.variant-gray {
+    background-color: #667c891a;
```

Comment:
I've been thinking about it myself :) 
In the end I came to the decision that since this color is not used anywhere else and is unlikely to be used anywhere else, it makes no sense to put them into variables